From 4f9222370cc974e68bc8dd7be9b93395b43d92e2 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Fri, 1 Jul 2022 11:10:48 +0200
Subject: [PATCH 0157/1141] arm64: defconfig: Enable the STM32MP25 PCIe
 RootComplex mode

PCIe controller Based on DesignWare core and STM32 ComboPHY to generate
the refClk.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I564b8f91ba87e23300af48f3501d151bb94ea8b0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/306766
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -13,6 +13,7 @@ CONFIG_NET_CLS_U32=m
 CONFIG_NET_SCH_MULTIQ=m
 CONFIG_NET_SCH_PRIO=m
 CONFIG_NET_SCH_SKBPRIO=m
+CONFIG_PCIE_STM32=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_REGULATOR_ARM_SCMI=y
